spec/models/article_spec.rb

```diff
@@ -138,7 +138,7 @@
         end
         params = { category_slug: 'category_2' }
         records = portal_2.articles.search(params)
-        expect(records.count).to eq(25)
+        expect(records.count).to eq(31)
```

Comment:
Having a random number here would be pretty confusing and error prone in the specs. I.e. how the number `31` is derived or why did we change the number 25.    

Either let's create a new category within this specific case, build a list of articles associated with that category and ensure that the specific count is resolved within this block. or at least lets add comments on how the number 31 is derived

Reply:
Let me add added articles_count here.